src/config/webpack.base.ts

```diff
@@ -132,7 +132,7 @@ export default ({ mode, paths, env, sourceMaps }: IWebpackConfig): Configuration
             {
               test: /(js|jsx|ts|tsx)$/,
               // include: paths.srcPaths,
-              exclude: /node_modules/,
+              // exclude: /node_modules/,
```

Comment:
can we explicitly include slate instead of including all of `node_modules`? i'm assuming this is going to tank build time since webpack has to pull in every single dep

Reply:
yeah, that sounds good to me!